Tooltip fixes

Arular101 opened the pull request:
Added to fake structures with a build limit of one that only one can be build.
Changed the transport helicopter husk names to Chinook. (Transport helicopter was changed to Chinook last release.)
Changed Mig to MiG. MiG is a abbreviation of Mikoyan and Gurevich, see [Wikipedia](https://en.wikipedia.org/wiki/Russian_Aircraft_Corporation_MiG).
Rearranged the support powers of the airfield to mirror the arrangement of the support powers in the upper left corner.
Removed "Can detect cloaked units." from SAM and AA Gun. (This feature was removed in the last release.)
Clarified some descriptions, please see the "Files changed".

I also wanted to add the description of the parabombs support power in the airfield for Ukraine only. But, I don't know if that's worth it.

I have two related questions:
1. Some units with no capabilities to shoot at aircraft has the description "Weak vs Aircraft". Also, the SAM and AA Gun has the description "Weak vs Ground units". Is this desired?
![2](https://user-images.githubusercontent.com/33390659/32942742-311500de-cb8a-11e7-8a53-9ffb338d831f.png)

2. Punctuation is not consistent. Is there any rule about how to do it?
![1](https://user-images.githubusercontent.com/33390659/32942853-8ede53a0-cb8a-11e7-8ef3-85bb4d1a293c.png)

Arular101 committed:
Tooltip fixes

reaperrr commented:
To 1., yes, iirc that's intentional.

> Punctuation is not consistent. Is there any rule about how to do it?

Not really, I think. Though by the look of things, I'd go for "Descriptive sentence should always end with full stop", "Strong/Weak vs. should always end *without* full stop".
It's up to you whether you add another commit here or file another PR later.

MustaphaTR reviewed (approved):
Current changes look good to me.

Arular101 committed:
More Changes

Arular101 commented:
Updated.

In the second commit I moved “Maximum 1 can be trained.” up to the other sentences without the two spaces, see:
![3](https://user-images.githubusercontent.com/33390659/32969790-ab61c9a6-cbe6-11e7-873f-7b310e694b4e.png)

There was luckily only one instance with wrong punctuation. 

Capitalized Tesla because it is a name.

Radar dome was the only structure with two spaces in front of the “Requires power to operate.”

I also wanted to remove the name of the unit in the description. It was only in three cases, these are: Light tank, Demolition truck and Chrono tank.

Added info for the Ranger that it can carry a passenger. Mammoth tank for crushing concrete walls.

The rest are minor adjustments. If something is not desired, or could be changed in a better way, please feel free to say it and I will gladly change it.

I’ll squash the commits after the review.

MustaphaTR commented:
Can you add `Can detect cloaked units.` to Tanya too?

Arular101 commented:
Good catch, will do.  I just added that for the Mammoth tank.

Arular101 commented:
Updated with Tanya can detect cloak.

MustaphaTR commented:
My 👍 still stands.

reaperrr commented:
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/6e565f54fd48a0f3e0204a73e1a03d060b99bd26...cf79687f14324135bcbe94f9d1cbf85306f03d1a)
